templates/base/head.tmpl

```diff
@@ -127,6 +127,9 @@
 	<meta property="og:url" content="{{AppUrl}}" />
 	<meta property="og:description" content="{{MetaDescription}}">
 {{end}}
+{{if not (eq DefaultTheme "default")}}
```

Comment:
Use ˋneqˋ instead

Reply:
Thanks @JonasFranzDEV. I've made the change.

Reply:
@JonasFranzDEV per the failed CI `neq` is not defined, and so I will revert back to original